pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[project]
+name = "neural-lam"
+version = "0.1.0"
+description = "LAM-based data-driven forecasting"
+authors = [
+    {name = "Joel Oskarsson", email = "joel.oskarsson@liu.se"},
+    {name = "Simon Adamov", email = "Simon.Adamov@meteoswiss.ch"},
+    {name = "Leif Denby", email = "lcd@dmi.dk"},
+]
+
+# PEP 621 project metadata
+# See https://www.python.org/dev/peps/pep-0621/
+dependencies = [
+    "numpy>=1.24.2",
+    "wandb>=0.13.10",
+    "scipy>=1.10.0",
+    "pytorch-lightning>=2.0.3",
+    "shapely>=2.0.1",
+    "networkx>=3.0",
+    "Cartopy>=0.22.0",
+    "pyproj>=3.4.1",
+    "tueplots>=0.0.8",
+    "matplotlib>=3.7.0",
+    "plotly>=5.15.0",
+    "torch>=2.3.0",
+    "torch-geometric==2.3.1",
+]
+requires-python = ">=3.9"
+
+[tool.pdm.dev-dependencies]
+dev = [
+    "pre-commit>=2.15.0",
+    "pytest>=8.2.1",
+    "pooch>=1.8.1",
+]
```

Comment:
Change to optional dependency group called just `dev`, to allow for installing both using pdm and pip.

Reply:
All done! :)